### ISSUE TITLE
Jaina Proudmore Not Present in SW: Unable to do quest; Fate, Up Against Your Will

### ISSUE THREAD

**Issue by Amplefere** [dalaranwow/dalaran-wow#4959]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Quest: Fate, Up Against Your Will
Unable to talk to Jaina Proudmore in Stormwind to continue quest as she is not there

**Expected behaviour**: Tell us what should happen instead
Jaina Proudmore should be Present Next to the King and should be able to interact with to continue quest and quest chain
**Steps to reproduce the problem**:

1. Start quest
2. "Unable to talk to Jaina as she is not there"
3. 

**Include proofs for this behaviour**
http://wowwiki.wikia.com/wiki/Quest:Fate,_Up_Against_Your_Will
https://www.youtube.com/watch?v=yXnAZZC6OTU

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Derolana** [comment]
did you complete all Dragonblight quests ?
http://www.wowhead.com/achievement=547/veteran-of-the-wrathgate after this you are able to pick quest from Alexstrasza and she stand infront of wrathgate - http://www.wowhead.com/quest=13347/reborn-from-the-ashes that send you to http://www.wowhead.com/quest=13369/fate-up-against-your-will

**Comment by Derolana** [comment]
If Jaina is not there this means you havent completed previous quests.
 http://www.wowhead.com/achievement=35/might-of-dragonblight and leading you to http://www.wowhead.com/quest=12499/return-to-angrathar ( you get the achie http://www.wowhead.com/achievement=547/veteran-of-the-wrathgate ), after this you are able to talk to Alextrasza( she stands infront of wrathgate ) and get the quest http://www.wowhead.com/quest=13347/reborn-from-the-ashes  ( end Stormwind) - the next quest is http://www.wowhead.com/quest=13369/fate-up-against-your-will .
im not sure this is issue

**Comment by Amplefere** [comment]
I have completed all prior quests ad I do have the achievement. Jaina is
supposed to always be standing next to the king in SW as it is not a phased
zone. I wrote this report because she is not there and i cannot complete
the quest that i am currently on.

**Comment by TeridanDW** [comment]
Hello and thank you for the comments @all.
The alliance version of the battle of undercity is not ready, that's why Jaina is still missing.
When the event is realeased Jaina will show up.

**Comment by Derolana** [comment]
@TeridanDW but i did it .

**Comment by Amplefere** [comment]
I have completed ALL prerequisite quests, picked up the quest that had me
bring the shield from the wrathgate to the king in SW. The following quest
that you get from the king, which requires you to talk to Jaina (who should
already be standing next to the king). Review the youtube vid i linked in
the "Proofs" and this should be obvious.
